Frameworks/libmysqlclient/build-libmysqlclient.sh

```
@@ -63,18 +63,18 @@ fi
 rm -rf $BUILD_DIR/arm64
 mkdir -p $BUILD_DIR/arm64
 /opt/homebrew/bin/brew install icu4c googletest bison flex ninja cmake lz4 zlib llvm openssl@3
-export MACOSX_DEPLOYMENT_TARGET=12.0
+export MACOSX_DEPLOYMENT_TARGET=13.0
 export OPENSSL_ROOT_DIR=$(/opt/homebrew/bin/brew --prefix openssl@3)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)